File: Python/aumento.py
salario = float(input("Digite o salario da pessoa: "))

if salario <= 1000.0:
    porcentagem = 20
elif salario <= 3000.0:
    porcentagem = 15
elif salario <= 8000.0:
    porcentagem = 10
else:
    porcentagem = 5

aumento = salario * porcentagem / 100
novoSalario = salario + aumento

print(f"Novo salario = R$ {novoSalario:.2f}")
print(f"Aumento = R$ {aumento:.2f}")
print(f"Porcentagem {porcentagem} %")

File: C/aumento.c
#include <stdio.h>

int main()
{
    double salario, aumento, novoSalario;
    int porcentagem;

    printf("Digite o salario da pessoa: ");
    scanf("%lf", &salario);

    if (salario <= 1000.0){
        porcentagem = 20;
    } else if (salario <= 3000.0){
        porcentagem = 15;
    } else if (salario <= 8000.0){
        porcentagem = 10;
    } else {
        porcentagem = 5;
    }

    aumento = (salario * porcentagem) / 100;
    novoSalario = salario + aumento;

    printf("Novo salario = R$ %.2lf\n", novoSalario);
    printf("Aumento = R$ %.2lf\n", aumento);
    printf("Porcentagem = %d %%", porcentagem);

    return 0;
}
